source/slurm/overview.rst

```diff
@@ -425,7 +425,7 @@ task/thread per node.
 .. note::
 
    You **must** specify a number of tasks, either with
-   ``-n`` (--ntasks) or ``-N`` (--nodes) or both. If you do not specify
+   ``-n`` (- -ntasks) or ``-N`` (--nodes) or both. If you do not specify
```

Comment:
Is this space intentional?  (I would think the double dash notation is correct)

Reply:
Per discussion on our call, it may have been some weird "em dash" conversion issue.  Try changing this to the below:

``-n`` (``--ntasks``) or ``-N`` (``--nodes``)